### ISSUE TITLE
App Support

### ISSUE THREAD

**Issue by rslueb** [beardedspice/beardedspice#930]
MyTuner

https://mytuner-radio.com

**Comment by martin-braun** [comment]
Probably the best radio app if you don't need to enter custom URLs. And if you do so, they might already have your favorite stream in their list. I wish this would be supported by BeardedSpice.

**Comment by jonbstrong** [comment]
Another vote for mytuner!

**Comment by garyking** [comment]
Yes, please support the [Mac app of myTuner](https://apps.apple.com/us/app/mytuner-radio-live-fm-stations/id881415018?mt=12).
